go/service/account.go

```diff
@@ -165,3 +165,32 @@ func (h *AccountHandler) SetLockdownMode(ctx context.Context, arg keybase1.SetLo
 	_, err = mctx.G().API.Post(mctx, apiArg)
 	return err
 }
+
+func (h *AccountHandler) PassphraseCheck(ctx context.Context, arg keybase1.PassphraseCheckArg) (ret bool, err error) {
```

Comment:
I think this should be in `engine` package.  Might make your testing easier too.

Reply:
done, can you take a look? hopefully it's not too rough, half way in the code i realised this is essentially wrapping `libkb.VerifyPassphraseForLoggedInUser` call in an RPC, and maybe engine is a bit of an overkill... but still